### PR TITLE
Updated Scatter graph documentation and examples to be carbon-centric.

### LINKED ISSUE #172: Update documentation for Scatter component to be carbon-centric
# Story

## Description

As a current consumer of carbon, I'd like for the public facing documentation for the Scatter component to utilize carbon until terra-graphs is released

## Motivation

So that I will have the information I need to use the Scatter component from carbon as things currently are released

## Definition of Done

We'll know we're done when the documentation in terra-graphs-docs is utilizing carbon instead of terra-graphs.

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
### Summary

- Made changes to the documentation for all Scatter graph examples to make the dev site carbon-centric.

Closes #172 

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/ScatterGraph/Reflow/DynamicData.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    graph.reflowMultipleDatasets(newDataset);
+
+    return {
+      initial: hour,
+      factor: panState.factor,
+    };
+  };
+
+  const [, dispatch] = React.useReducer(reducer, state);
```
Comment (tbiethman) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/ScatterGraph/Reflow/DynamicData.jsx:57]: Same comment here regarding reducer side-effects: https://github.com/cerner/terra-graphs/pull/193/files#r679991504

Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/ScatterGraph/Reflow/DynamicData.jsx]: Fixed [here](https://github.com/cerner/terra-graphs/pull/194/commits/0e04dfd7c03b7659f8a588266b0f6ce7b7b43f07).

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/ScatterGraph/Reflow/SimplePanning.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      case 'panLeft':
+        hour = panState.initial - panState.factor;
+        break;
+      case 'panRight':
+        hour = panState.initial + panState.factor;
+        break;
+    }
+
+    graph.config.axis.x.lowerLimit = new Date(2016, 0, 1, hour).toISOString();
+    graph.config.axis.x.upperLimit = new Date(2016, 0, 2, hour).toISOString();
```
Comment (tbiethman) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/ScatterGraph/Reflow/SimplePanning.jsx:43]: Can remove 42, 43 now

Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/ScatterGraph/Reflow/SimplePanning.jsx]: Yeah realized lately and removed it in the next [commit](https://github.com/cerner/terra-graphs/pull/194/commits/91e4839e0402cc8fd67ea792f3efc6c6bfef4860#diff-51aa3df6b894082f674c1535b2e1fe854986b25c285136985e89111ccb86d699L42).

**Review by tbiethman — APPROVED** [review]

**Review by sdadn — APPROVED** [review]

**Pull request merged**